src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -155,6 +156,13 @@ namespace AppInstaller::CLI::Execution
         }
     }
 
+    int Reporter::PromptForAnyKey(Resource::LocString message, Level level)
+    {
+        auto out = GetOutputStream(level);
+        out << message << std::endl;
+        return _getch();

```

Comment:
The field `std::istream& m_in` has to be used for all input.  I think that despite the documentation, the buffer inside the standard input stream waits for characters to be present rather than returning errors.  From looking at the code, I don't see why `m_in.get()` wouldn't work here.

Reply:
It would work; I'll just have to change it to be `PromptForEnter` since buffered input streams won't return unless enter is pressed.